pydantic/import_helper.py

```diff
@@ -0,0 +1,382 @@
+import warnings
+from importlib import import_module
+from types import ModuleType
+from typing import Any, Tuple
+
+
+class V2MigrationRemovedException(Exception):
+    ...
+
+
+class V2MigrationMovedWarning(DeprecationWarning):
+    ...
+
+
+class V2MigrationRenamedWarning(DeprecationWarning):
+    ...
+
+
+class V2MigrationSuperseded(Exception):
+    ...
+
+
+V2_MIGRATION_MAPPING = {
+    # Case: Removed
+    'pydantic.Required': V2MigrationRemovedException('pydantic.Required has been removed in favour of ...'),
```

Comment:
looking good. Did you use a script to find/generate these? If so might be worth including it in the PR in case it's needed again.

Reply:
Not that sophisticated 😄 gave it a go, but didnt want to sink too much time into it... So I just looked at the git diff between each of the versions.